examples/ats-keyword/.github/workflows/avh.yml

```diff
@@ -1,79 +0,0 @@
-# This is a basic workflow to help you get started with Actions on CMSIS projects
-
-name: ATS keyword workflow
```

Comment:
why was this removed?  how does github perform its CI?

Reply:
Yes, we need github CI be added separately. The upstream project has it removed as it's not being hosted on github (this config is invalid).

The team will add github CI in separate pull request.